core/src/test/java/org/frankframework/filesystem/smb/Samba2FileSystemTest.java

```diff
@@ -69,6 +70,21 @@ public void basicFileSystemTestExists() throws Exception {
 		super.basicFileSystemTestExists();
 	}
 
+	@Test
+	void fileSystemTestAfterClosingAndOpening() throws Exception {
+		// Arrange & Assert 1
+		super.basicFileSystemTestExists();
+		fileSystem.deleteFile(fileSystem.toFile("testExists" + FILE1));
+		assertFalse(fileSystem.exists(fileSystem.toFile("testExists" + FILE1)));
+
+		// Close & Open Samba FS
+		fileSystem.close();
+		fileSystem.open();
+
+		// Arrange & Assert 2
+		super.basicFileSystemTestExists();
```

Comment:
Ik liep er bij mij tegen aan (toe ik deze verplaatst had) dat in de delegated methode hij ook al open aanroept.

Reply:
Ja, kan idd weg. Aangepast.